apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -63,13 +63,10 @@ loop(Socket, Transport, State) ->
                     InfoBin = dproto_tcp:encode_bucket_info(Info),
                     Transport:send(Socket, InfoBin),
```

Comment:
Good idea, I see there is already an entry and I'm trying to figure out why it's not taking effect:
https://github.com/dataloop/dalmatinerdb/blob/master/elvis.config#L21-L24